modules/statistics/php/stats_mri.class.inc

```diff
@@ -334,18 +341,6 @@ class Stats_MRI extends \NDB_Form
         } else {
             $MRI_Type = $scan_types[$_REQUEST['mri_type']];
         }
-        //TODO IBIS
```

Comment:
@sruthymathew123 you should know about these for IBIS

Reply:
@ridz1208 Thanks Rida. But for some reason IBIS is no more using imaging statistics.